addons/l10n_za/data/l10n_za_chart_data.xml

```diff
@@ -0,0 +1,192 @@
+            
+        </record>
+
+        <record id="200060"  model="account.account.template">
+            <field name="code">200060</field>
+            <field name="name">Sales Tax Control Account </field>
+            <field name="reconcile" eval='False'/>
+            <field name="user_type_id" ref="account.data_account_type_current_liabilities"/>
+            
+        </record>
+
+        <record id="100060"  model="account.account.template">
+            <field name="code">100060</field>
+            <field name="name">Purchase Tax Control Account </field>
+            <field name="reconcile" eval='False'/>
+            <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+            
+        </record>
+
+        
+        <record id="l10n_za" model="account.chart.template">
+            <field name="name">South African Tax and Account Chart Template (by Paradigm Digital)</field>
+            <field name="bank_account_code_prefix">1200</field>
+            <field name="cash_account_code_prefix">1250</field>
+            <field name="code_digits">6</field>
+            <field name="currency_id" ref="base.ZAR"/>
+            <field name="transfer_account_id" ref="transfer_account"/>
+        </record>
+
+        <record id="transfer_account" model="account.account.template">
+            <field name="chart_template_id" ref="l10n_za"/>
+        </record>
+
+        <record id="110010"  model="account.account.template">
+            <field name="chart_template_id" ref="l10n_za"/>
+        </record>
+
+        <record id="220010"  model="account.account.template">
+            <field name="chart_template_id" ref="l10n_za"/>
+        </record>
+
+        <record id="600010"  model="account.account.template">
+            <field name="chart_template_id" ref="l10n_za"/>
+        </record>
+
+        <record id="500010"  model="account.account.template">
+            <field name="chart_template_id" ref="l10n_za"/>
+        </record>
+
+        <record id="200010"  model="account.account.template">
+            <field name="chart_template_id" ref="l10n_za"/>
+        </record>
+
+        <record id="100050"  model="account.account.template">
+            <field name="chart_template_id" ref="l10n_za"/>
+        </record>
+
+        <record id="100020"  model="account.account.template">
+            <field name="chart_template_id" ref="l10n_za"/>
+        </record>
+
+        <record id="610440"  model="account.account.template">
+            <field name="chart_template_id" ref="l10n_za"/>
+        </record>
+
+        <record id="200060"  model="account.account.template">
+            <field name="chart_template_id" ref="l10n_za"/>
+        </record>
+
+        <record id="100060"  model="account.account.template">
+            <field name="chart_template_id" ref="l10n_za"/>
+        </record>
+
+        <record id="ST1" model="account.tax.template">
```

Comment:
I had a case of requiring to update the data (maybe a related field was missing) - I will try to create in one step.

Reply:
Important Accounts that are referenced in the chart template below. While less important accounts/taxes are in separate files.

Reply:
@h4818 Normally the chart template is defined twice.  Once without accounts and then with.  This way you don't have to redefine accounts.  But, it is something we can easily adapt ourselves.

Reply:
Yes, the chart has to be created before it can be used in the account definition. Hence multiple chart/account definitions. I guess this could be cleaner :)